README.md

```diff
@@ -925,6 +925,59 @@ Back endpoint responsible for fetching and sending information to the database r
 `api/wellness` Sends an answer boolean to the database that specifies whether a student is symptomatic or not: true = symptomatic, false = not symptomatic.
 
 
+### Jobs
+Back endpoint responsible for fetching and sending information to the database regarding Jobs and timesheet used to clock shifts.
+
+##### GET 
+
+`api/jobs/getJobs` Gets the users current active jobs and returns them in a list.  
+
+`api/jobs/getSavedShifts/` Gets the users active shifts that have yet to be approved.
+
+`api/jobs/supervisorName/{supervisorID}`Gets the name of the supervisors associated with the user ID numbers.
+
+`api/jobs/clockOut` Gets the current status of the users shift from the back end. returns bool of status along with timestamp.
+
+`api/jobs/canUsePage` Gets the status of the user's account. if returns anything, user is a staff, Else user is a student.
```

Comment:
```suggestion
`api/jobs/isStaff` Gets the status of the user's account for jobs purposes.  A non-empty response means the user has staff job(s), and an empty response means the user has student job(s).  This is important because, for example, a staff employee may also be taking a class and therefore also considered a student.  This helps with other special cases, too.
```